plugins/docs-tools/skills/docs-workflow-create-jira/scripts/create-jira-ticket.sh

```
@@ -15,8 +15,9 @@ PLAN_FILE="${3:?Missing PLAN_FILE argument}"
 
 SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
 
-# Source ~/.env for credential mapping
+# Load global defaults, then local overrides
 source ~/.env 2>/dev/null || true
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)